services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/VerificationPacket.java

```diff
@@ -185,76 +111,4 @@ public IPacket deserialize(byte[] data, int offset, int length) {
         } while (tlv.getType() != 0 && bb.hasRemaining());
         return this;
     }
-
-    /* (non-Javadoc)
-     * @see java.lang.Object#hashCode()
-     */
-    @Override
-    public int hashCode() {
-        final int prime = 883;
-        int result = super.hashCode();
-        result = prime * result
-                + ((chassisId == null) ? 0 : chassisId.hashCode());
-        result = prime * result + (optionalTlvList.hashCode());
-        result = prime * result + ((portId == null) ? 0 : portId.hashCode());
-        result = prime * result + ((ttl == null) ? 0 : ttl.hashCode());
-        return result;
-    }
-
-    /* (non-Javadoc)
-     * @see java.lang.Object#equals(java.lang.Object)
-     */
-    @Override
-    public boolean equals(Object obj) {
-        if (this == obj) {
-            return true;
-        }
-        if (!super.equals(obj)) {
-            return false;
-        }
-        if (!(obj instanceof VerificationPacket)) {
-            return false;
-        }
-        VerificationPacket other = (VerificationPacket) obj;
-        if (chassisId == null) {
-            if (other.chassisId != null) {
-                return false;
-            }
-        } else if (!chassisId.equals(other.chassisId)) {
-            return false;
-        }
-        if (!optionalTlvList.equals(other.optionalTlvList)) {
-            return false;
-        }
-        if (portId == null) {
-            if (other.portId != null) {
-                return false;
-            }
-        } else if (!portId.equals(other.portId)) {
-            return false;
-        }
-        if (ttl == null) {
-            if (other.ttl != null) {
-                return false;
-            }
-        } else if (!ttl.equals(other.ttl)) {
-            return false;
-        }
-        return true;
-    }
-
-    @Override
-    public String toString() {
```

Comment:
Are you sure we don't use that in log files and kibana dashboards?

Reply:
I didn't found any place where we are using Verification Packet in logs. So I guess it wouldn't effect on Kibana